plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -699,6 +702,7 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
             protocol_features_dir = app().config_dir() / pfd;
          else
             protocol_features_dir = pfd;
+         resource_monitor_plugin::monitor_directory(protocol_features_dir);
```

Comment:
Do we write anything to `protocol_features_dir` after startup? I don't think we need to monitor it.

Reply:
Will remove.